configuretools.cmake

```
@@ -1,6 +1,6 @@
 # Get the version of the compiler that is in the file name for tool location.
 set (CLR_CMAKE_COMPILER_FILE_NAME_VERSION "")
-if (CMAKE_C_COMPILER MATCHES "-?[0-9]+\.?[0-9]+$")
+if (CMAKE_C_COMPILER MATCHES "-?([0-9]+\.)?[0-9]+$")
```

I'd prefer if we made the second part of the regex optional instead of the first part. I think it makes it easier to read.

```suggestion
if (CMAKE_C_COMPILER MATCHES "-?[0-9]+(\.[0-9]+)?$")
```

---

Makes sense